### DISCUSSION TITLE
GoReleaser build not triggered

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/goreleaser#2289] [Q&A] [answered]
Following up on https://github.com/goreleaser/goreleaser/discussions/2247, I'm facing another case that gorelease build is not triggered, and I've ran out of the ideas why. 

- My [go-release-build.yml](https://github.com/go-easygen/wireframe/blob/master/.github/workflows/go-release-build.yml) file, it is the same as other working files
- I've [pushed a new tag](https://github.com/go-easygen/wireframe/releases/tag/v1.0.2) alone the changes
- but [only the master branch was built](https://github.com/go-easygen/wireframe/actions), nothing for my  [pushed new tag](https://github.com/go-easygen/wireframe/releases/tag/v1.0.2)

I've ran out of the ideas why it is like this.  
Please help. thx.

**Comment by caarlos0** [comment, marked as answer]
Seems to be an issue in github actions, not goreleaser...

In any case, I looked and don't know what the problem is as well...

**Reply by suntong** [reply]
Ok, at least I'm not missing anything. 

Alright. Thanks for checking. I'll dig more ... and report back if I found anything...

**Reply by suntong** [reply]
Indeed its' an issue in github actions -- I made some irrelevant changes and force push just now. 
GoReleaser build is now triggered, without me doing actually anything.

**Reply by caarlos0** [reply]
nice! :)
